frontend/src/utils/sessionStorage/localSsotController/ssot.js

```diff
@@ -23,22 +23,34 @@ import {
   deleteUnusedAttributesFromDB,
 } from './attributes';
 
-const ROBOT_ID_PATH = 'robotId';
+const ROBOT_METADATA_PATH = 'robotMetadata';
 const ATTRIBUTE_STORAGE_PATH = 'attributeLocalStorage';
 const PARAMETER_STORAGE_PATH = 'parameterLocalStorage';
 
 /**
  * @description Gets the robotId of the currently opened robot from the session storage
  * @returns Currently saved robotId
  */
-const getRobotId = () => JSON.parse(sessionStorage.getItem(ROBOT_ID_PATH));
+const getRobotId = () =>
+  JSON.parse(sessionStorage.getItem(ROBOT_METADATA_PATH)).robotId;
 
 /**
- * @description Writes the robotId of the currently opened robot into the session storage
+ * @description Gets the robotName of the currently opened robot from the session storage
+ * @returns Currently saved robotName
```

Comment:
Can you add a type here? I guess `{String}` is the type, isn't it?

Reply:
fixed